services/configServices/__tests__/NetlifyTomlService.js

```diff
@@ -0,0 +1,54 @@
+const axios = require("axios")
```

Comment:
rename the file to include `.spec`

Reply:
Good catch, [a6cdec7](https://github.com/isomerpages/isomercms-backend/pull/357/commits/a6cdec7b64e02ef1769b77c35a2cfcfa3b882986)